protos/flyteidl/admin/project.proto

```diff
@@ -22,6 +31,8 @@ message Project {
     repeated Domain domains = 3;
 
     string description      = 4;
+
+    repeated Tag tags = 5;
```

Comment:
how about we re-use - https://github.com/lyft/flyteidl/blob/master/protos/flyteidl/admin/common.proto#L227